scripts/chickens.lua

```diff
@@ -0,0 +1,149 @@
+
+		Turn(body, z_axis, math.rad(5), math.rad(20))
+		Turn(lthigh, z_axis, math.rad(-5), math.rad(20))
+		Turn(rthigh, z_axis, math.rad(-5), math.rad(20))
+		Move(body, y_axis, 0.7, 4000)			
+		Turn(tail, y_axis, math.rad(10), math.rad(40))
+		Turn(head, x_axis, math.rad(-10), math.rad(20))
+		Turn(tail, x_axis, math.rad(10), math.rad(20))
+		WaitForTurn(lthigh, x_axis)
+
+		Turn(lthigh, x_axis, math.rad(-10), math.rad(160))
+		Turn(lknee, x_axis, math.rad(15), math.rad(145))
+		Turn(lshin, x_axis, math.rad(-60), math.rad(250))
+		Turn(lfoot, x_axis, math.rad(30), math.rad(145))
+
+		Turn(rthigh, x_axis, math.rad(40), math.rad(145))
+		Turn(rknee, x_axis, math.rad(-35), math.rad(145))
+		Turn(rshin, x_axis, math.rad(-40), math.rad(145))
+		Turn(rfoot, x_axis, math.rad(35), math.rad(145))
+
+		Move(body, y_axis, 0, 4000)
+		Turn(head, x_axis, math.rad(10), math.rad(20))
+		Turn(tail, x_axis, math.rad(-10), math.rad(20))
+		WaitForTurn(lshin, x_axis)
+
+		Turn(rthigh, x_axis, math.rad(70), math.rad(115))
+		Turn(rknee, x_axis, math.rad(-40), math.rad(145))
+		Turn(rshin, x_axis, math.rad(20), math.rad(145))
+		Turn(rfoot, x_axis, math.rad(-50), math.rad(210))
+
+		Turn(lthigh, x_axis, math.rad(-20), math.rad(210))
+		Turn(lknee, x_axis, math.rad(-60), math.rad(210))
+		Turn(lshin, x_axis, math.rad(50), math.rad(210))
+		Turn(lfoot, x_axis, math.rad(30), math.rad(210))
+
+		Turn(tail, y_axis, math.rad(-10), math.rad(40))
+		Turn(body, z_axis, math.rad(-5), math.rad(20))
+		Turn(lthigh, z_axis, math.rad(5), math.rad(20))
+		Turn(rthigh, z_axis, math.rad(5), math.rad(20))
+		Move(body, y_axis, 0.7, 4000)
+		Turn(head, x_axis, math.rad(-10), math.rad(20))
+		Turn(tail, x_axis, math.rad(10), math.rad(20))
+		WaitForTurn(rthigh, x_axis)
+
+		Turn(rthigh, x_axis, math.rad(-10), math.rad(160))
+		Turn(rknee, x_axis, math.rad(15), math.rad(145))
+		Turn(rshin, x_axis, math.rad(-60), math.rad(250))
+		Turn(rfoot, x_axis, math.rad(30), math.rad(145))
+
+		Turn(lthigh, x_axis, math.rad(40), math.rad(145))
+		Turn(lknee, x_axis, math.rad(-35), math.rad(145))
+		Turn(lshin, x_axis, math.rad(-40), math.rad(145))
+		Turn(lfoot, x_axis, math.rad(35), math.rad(145))
+
+		Move(body, y_axis, 0, 4000)
+		Turn(head, x_axis, math.rad(10), math.rad(20))
+		Turn(tail, x_axis, math.rad(-10), math.rad(20))
+		WaitForTurn(rshin, x_axis)
+	end
+end
+
+function script.StopMoving()
+	Signal(SIG_MOVE)
+	Turn(lfoot, x_axis, 0, math.rad(200))
+	Turn(rfoot, x_axis, 0, math.rad(200))
+	Turn(rthigh, x_axis, 0, math.rad(200))
+	Turn(lthigh, x_axis, 0, math.rad(200))
+	Turn(lshin, x_axis, 0, math.rad(200))
+	Turn(rshin, x_axis, 0, math.rad(200))
+	Turn(lknee, x_axis, 0, math.rad(200))
+	Turn(rknee, x_axis, 0, math.rad(200))
+end
+
+local function RecoilThread()
+	Turn(lsack, y_axis, math.rad(40), math.rad(440)) -- this had to be changed because it looked bad
+	Turn(rsack, y_axis, math.rad(-40), math.rad(440)) -- ditto. 1 isn't fast enough!
```

Comment:
Stick to the original values?
```suggestion
	Turn(lsack, y_axis, math.rad(40), math.rad(360))
	Turn(rsack, y_axis, math.rad(-40), math.rad(360))
```